frontend/src/store/Post/postInterfaces.ts

```diff
@@ -91,6 +91,12 @@ export interface PlaceType {
   phone_number: string;
   address: string;
   category: string;
+  index?: number;
+}
+
+export interface PlaceDayType {
+  place: PlaceType;
+  day?: number | string | null;
```

Comment:
Oh here are the types of `day`. Is the `null` type necessary? I think it can be avoided.

Reply:
While implementing dnd action, I united 1) places in cart section 2) places in my route section with one type, which is PlaceDayType. I differentiated them with its day value. Places in cart section has null value. Once they are dragged into my route section, selectedDay will override its value. To describe this, i added null type there!

Reply:
Oh, I see your point! Then how about using 0? In my opinion, limiting type as strictly as possible is better. But It is totally up to you. I will approve this PR regardless of the type.

Reply:
Having day value as 0 seems to be the nicest solution! I will correct its type and merge to the main branch. thanks for the review :)